go.mod

```diff
@@ -5,29 +5,33 @@ go 1.20
 require (
 	github.com/dgraph-io/badger/v3 v3.2103.5
 	github.com/gofiber/fiber/v2 v2.42.0
-	github.com/hashicorp/go-hclog v0.9.1
+	github.com/hashicorp/go-hclog v1.4.0
 	github.com/hashicorp/raft v1.3.11
-	github.com/hashicorp/raft-boltdb v0.0.0-20230125174641-2a8082862702
+	github.com/hashicorp/raft-boltdb/v2 v2.2.2
 	github.com/narvikd/errorskit v1.0.0
 	github.com/narvikd/fiberparser v1.1.1
 	github.com/narvikd/filekit v1.0.1
+	github.com/prometheus-community/pro-bing v0.1.0
+	google.golang.org/grpc v1.53.0
+	google.golang.org/protobuf v1.28.1
 )
 
 require (
 	github.com/andybalholm/brotli v1.0.4 // indirect
-	github.com/armon/go-metrics v0.3.8 // indirect
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878 // indirect
```

Comment:
Isn't it weird that before these changes go metrics was 0.38 and now it's a non-version?

Reply:
This is being pulled by raft boldb v2: https://github.com/hashicorp/raft-boltdb/blob/2a80828627023c0835e68f992ea082a26508037b/v2/go.mod#L6

It's only metrics, it's not a big deal.